roles/kolibri/defaults/main.yml

```diff
@@ -16,7 +16,7 @@
 # https://github.com/iiab/iiab/issues/1675
 # https://github.com/learningequality/kolibri/issues/5664
 
-kolibri_deb_url: https://learningequality.org/r/kolibri-deb-latest
+kolibri_deb_url: http://ppa.launchpad.net/learningequality/kolibri/ubuntu/pool/main/k/kolibri-source/kolibri_0.12.9-0ubuntu2_all.deb
```

Comment:
Just FYI the ongoing situation with Kolibri 0.12.9 not installing is being monitored at #2045, with a safe fix (fallback to Kolibri 0.12.8) proposed at PR #2046.

Reply:
Above commit needs to be reverted given upstream fix landed, as mentioned in #2045.

Explanation from PR #2046 should be in-lined or added as a separate PR.

Reply:
I'm going to leave this one as is for now